test/Microsoft.TestPlatform.AcceptanceTests/DifferentTestFrameworkSimpleTests.cs

```diff
@@ -67,6 +70,7 @@ public void WebTestRunAllTestsWithRunSettings(RunnerInfo runnerInfo)
         }
 
         [TestMethod]
+        [TestCategory("Windows-Review")]
```

Comment:
What is difference between 'Windows' and 'Windows-Review' test category?

Reply:
Windows-Review is added in this PR, those are tests that run currently only on windows but should be reviewed again to make sure we are running all tests that can run on Linux.